systemd/emulation.go

```diff
@@ -39,18 +38,24 @@ type emulation struct {
 	rootDir string
 }
 
-var errNotImplemented = errors.New("not implemented in emulation mode")
+type errNotImplemented struct {
```

Comment:
(nitpick) I would now call it `notImplementedError`, the pattern in go seems to be that the `ErrXXXX` prefix is used for simple errors like `ErrWriteToConnected = errors.New(...)` and the `xxxxError` postfix when there are error structs. I vaguely remember that this was also written down somewhere but I can't find it anymore.

Reply:
Done.